src/main.ts

```diff
@@ -0,0 +1,78 @@
+import path from 'path'
+import * as core from '@actions/core'
+import * as exec from '@actions/exec'
+import * as tc from '@actions/tool-cache'
+import * as utils from './utils'
+
+function getArgs(flag: string, input: string): string[] {
+  const value = core.getInput(input)
+  if (value) return [flag, value]
+  return []
+}
+
+async function run(): Promise<void> {
+  const yorVersion = core.getInput('version')
+  const commitChanges = core.getInput('commit_changes') === 'YES'
+
+  // Computing args
+  const yorArgs: string[] = [
+    'tag',
+    getArgs('-d', 'directory'),
+    getArgs('--tag-groups', 'tag_groups'),
+    getArgs('--tag', 'tag'),
+    getArgs('--skip-tags', 'skip_tags'),
+    getArgs('--skip-dirs', 'skip_dirs'),
+    getArgs('--skip-resource-types', 'skip_resource_types'),
+    getArgs('--custom-tagging', 'custom_tags'),
+    getArgs('--output', 'output_format'),
+    getArgs('--config-file', 'config_file')
+  ].flat()
+
+  const logLevel = core.getInput('log_level')
+  if (logLevel) {
+    process.env['LOG_LEVEL'] = logLevel
```

Comment:
This not idiomatic. IMO, that shouldnt be an input, the user should just provide the environment variable

Reply:
Agreed, let's remove it